nextcloudappstore/core/static/assets/css/style.css

```diff
@@ -249,6 +249,7 @@ form .text-danger {
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
+    line-height: 1.2em;
```

Comment:
For me this does not change anything, but `1.3em` does fix it. (Safari on macOS)